src/website/views/components/DropDown/DropDown.module.css

```diff
@@ -0,0 +1,56 @@
+.flex-item-container {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  font-family: 'Switzer, sans-serif';
```

Comment:
Same issue - take a look at this link: [cdnfonts.com/switzer.font](https://www.cdnfonts.com/switzer.font) to use this font.